templates/field/field--node--oe-project-participants--oe-project.html.twig

```diff
@@ -0,0 +1,11 @@
+{#
+/**
+ * @file
+ * Project participant field template.
```

Comment:
The field name is "Participants | oe_project_participants", i would put "Project participants"